src/E1_process_results.py

```diff
@@ -605,7 +604,7 @@ def convert_components_to_dataframe(dict_values):
     df_comp = df_comp.reset_index()
 
     for i in range(len(df_comp)):
-        if df_comp.at[i, "Optimization"]:
+        if df_comp.at[i, "Capacity optimization"]:
```

Comment:
```suggestion
        if df_comp.at[i, "Capacity optimization"] is True:
```
Defensive programming